packages/forms/stories/15-Fieldset.stories.tsx

```diff
@@ -0,0 +1,157 @@
+} from '@zendeskgarden/react-forms';
+
+import { IFieldsetStoryProps, FIELDSET_ARGS, FIELDSET_ARGS_TYPES } from './story-types';
+
+export default {
+  title: 'Components/Forms/Fieldset',
+  component: Fieldset,
+  subcomponents: {
+    Fieldset,
+    'Fieldset.Legend': Fieldset.Legend
+  }
+} as Meta;
+
+export const RadioGroup: Story<IFieldsetStoryProps & InputHTMLAttributes<HTMLInputElement>> = ({
+  disabled,
+  validation,
+  isCompact,
+  isHidden,
+  showHint,
+  showMessage
+}) => {
+  return (
+    <Grid>
+      <Row>
+        <Col lg={4} offsetLg={4} md={6} offsetMd={3}>
+          <Fieldset isCompact={isCompact} disabled={disabled}>
+            <Fieldset.Legend hidden={isHidden}>Choose a growth type</Fieldset.Legend>
+            <Field>
+              <Radio value="annual" name="example">
+                <Label>Annual</Label>
+                {showHint && <Hint>Hint</Hint>}
+              </Radio>
+            </Field>
+            <Field>
+              <Radio value="perennial" name="example">
+                <Label>Perennial</Label>
+                {showHint && <Hint>Hint</Hint>}
+                {showMessage && <Message validation={validation}>Message</Message>}
+              </Radio>
+            </Field>
+          </Fieldset>
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+RadioGroup.args = {
+  ...FIELDSET_ARGS
+};
+
+RadioGroup.argTypes = {
+  ...FIELDSET_ARGS_TYPES
+};
+
+export const CheckboxGroup: Story<IFieldsetStoryProps & InputHTMLAttributes<HTMLInputElement>> = ({
+  disabled,
+  validation,
+  isCompact,
+  isHidden,
+  showHint,
+  showMessage
+}) => {
+  return (
+    <Grid>
+      <Row>
+        <Col lg={4} offsetLg={4} md={6} offsetMd={3}>
+          <Fieldset isCompact={isCompact} disabled={disabled}>
+            <Fieldset.Legend hidden={isHidden}>Choose growth types</Fieldset.Legend>
+            <Field>
+              <Checkbox>
+                <Label>Annual</Label>
+                {showHint && <Hint>Hint</Hint>}
+              </Checkbox>
+            </Field>
+            <Field>
+              <Checkbox>
+                <Label>Perennial</Label>
+                {showHint && <Hint>Hint</Hint>}
+                {showMessage && <Message validation={validation}>Message</Message>}
+              </Checkbox>
+            </Field>
+          </Fieldset>
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+CheckboxGroup.args = {
+  ...FIELDSET_ARGS
+};
+
+CheckboxGroup.argTypes = {
+  ...FIELDSET_ARGS_TYPES
+};
+
+export const ToggleGroup: Story<IFieldsetStoryProps & InputHTMLAttributes<HTMLInputElement>> = ({
+  disabled,
+  validation,
+  isCompact,
+  isHidden,
+  showHint,
+  showMessage
+}) => {
+  return (
+    <Grid>
+      <Row>
+        <Col lg={4} offsetLg={4} md={6} offsetMd={3}>
+          <Fieldset isCompact={isCompact} disabled={disabled}>
+            <Fieldset.Legend hidden={isHidden}>Toggle growth types</Fieldset.Legend>
+            <Field style={{ marginBottom: '4px' }}>
```

Comment:
Please remove this margin customization. We want the designers to review the default layout.